### PR TITLE
The description of pr is now editable

### LINKED ISSUE #343: Ability to edit the description of a pull request

### PR DESCRIPTION
**Pull request by eanker**
The description is editable
The revert edit mode icon (blue eye) has been replaced by a edit icon.
Fixes #343
(I got the error of groups again on our tool, so the pr is made but nothing more).

### CODE REVIEW

File: src/projects/pull-request/pull-request-general-information.html [hunk 1]
```diff
@@ -143,7 +150,7 @@
       </header>
       <section>
         <div class="main">
-          <marked-github-element markdown="[[pullRequest.body]]"></marked-github-element>
+          <markdown-input preview-toggle="{{shouldSend}}" markdown="{{pullRequest.body}}"></markdown-input>
```
Comment (txsmith) [inline src/projects/pull-request/pull-request-general-information.html]: The naming of the `shouldSend` property can be improved in my opinion

File: src/util/markdown-input.html [hunk 2]
```diff
@@ -73,8 +73,7 @@
       paper-icon-button {
         margin-top: 1em;
       }
-      paper-icon-button:hover,
-      paper-icon-button[active] {
```
Comment (txsmith) [inline src/util/markdown-input.html:77]: Why was this `active` selector removed?

**Comment by txsmith** [comment]
screenshot from 2016-07-04 13 02 18
I think the CSS should be adjusted such that the markdown input for the description aligns with the one for the comments

**Comment by txsmith** [comment]
Tested locally, confirmed working!

File: src/util/markdown-input.html (hunk 2 above)
Comment (eanker) [inline src/util/markdown-input.html:77]: Now we switch from view icon to draw icon and therefore the draw icon would always be blue.
This is replaced on all things, like the leave a comment.

**Comment by txsmith** [comment]
Looks good!

**Pull request merged**